### PR TITLE
[docs] how-to: run `actions` from RunJS

### PR DESCRIPTION
**Pull request by withshubh**
#### Changes
- Added how-to guide on triggering actions from RunJS query

### CODE REVIEW

**Comment by withshubh** [comment]
https://app.clickup.com/t/2hxxcdf

**Comment by shubh22** [comment]
Task linked: [CU-2hxxcdf Documentation for how actions can be run from RunJS](https://app.clickup.com/t/2hxxcdf)

File: docs/docs/how-to/run-action-from-runjs.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+![ToolJet - How To - Run Actions from RunJS query](/img/how-to/run-actions-from-runjs/closemodal.png)
+
+</div>
+
+### Set Local Storage
+
+**Syntax:**
+
+```javascript
+actions.setLocalStorage('key':'value')
```
Comment (stepinfwd) [inline docs/docs/how-to/run-action-from-runjs.md:111]: actions.setLocalStorage('key' , 'value')

**Review by stepinfwd — CHANGES_REQUESTED** [review]

**Comment by shubh22** [comment]
Task linked: [CU-2juq0q0 [docs] how-to: run `actions` from RunJS](https://app.clickup.com/t/2juq0q0)

**Pull request merged**